static/js/add_category.js

```diff
@@ -0,0 +1,12 @@
+$('#addcategory').validate({
+rules: {
+        name:{
+        required:true
+        }
+ },
+ messgaes: {
+    name:{
+    required:'please enter category name'
```

Comment:
@Yshah112 if you are writing a message or a sentence then please make sure it's initial is capital and add full stop.

cc: @lalitvasoya

Reply:
done